salsa_dancing_molecules/simulations/nve.py

```diff
@@ -79,7 +91,32 @@ def printenergy(a=atoms):
               'Etot = %.3feV' % (epot, ekin, ekin / (1.5 * units.kB),
                                  epot + ekin))
 
+        # Important Quantaties to save
+        potential_energies.append(epot)
```

Comment:
Replace printenergy with your own function in a separate module and encapsulate all the recording functionality there.